frontend/javascripts/oxalis/view/scalebar.js

```diff
@@ -58,10 +86,11 @@ function Scalebar({ zoomValue, dataset, widthInPixels, heightInPixels }: Props)
           position: "absolute",
           bottom: "1%",
           right: "1%",
-          // The scalebar should have a width of 25% from the actual viewport (without the borders)
-          width: `calc(25% - ${Math.round(
-            ((2 * OUTER_CSS_BORDER) / constants.VIEWPORT_WIDTH) * 100,
-          )}%)`,
+          width: collapseScalebar
+            ? 16
+            : `calc(${scaleBarWidthFactor * 100}% - ${Math.round(
+                ((2 * OUTER_CSS_BORDER) / constants.VIEWPORT_WIDTH) * 100,
+              )}%)`,
```

Comment:
I created a "horizontal" tree and tried to make it as large as the scalebar. When using the context menu on the tree and choosing "Path length ...", the length differed slightly. It looks like the written length represents the black box and not the the distance between the white lines.

![image](https://user-images.githubusercontent.com/2486553/153386692-c57d0b20-70ca-42dd-9847-159f5747f279.png)

In line 101, a padding of `4` is defined for this div. I suspect, that that padding has to be taken into account here. Can you extract the padding into a variable and add it to the calc expression? I think, the new expression would be:

```suggestion
          width: collapseScalebar
            ? 16
            : `calc(${scaleBarWidthFactor * 100}% - ${Math.round(
                ((2 * OUTER_CSS_BORDER) / constants.VIEWPORT_WIDTH) * 100,
              )}% + ${2 * PADDING}px)`,
```

Reply:
Nice catch 👀 Thanks!

Reply:
just fyi: pulling the padding into the term with the outer css border is not equivalent to my suggestion above. Since I still saw the imprecision problem, I just checked it again and the other version indeed works. so, everything should be fine now :)